certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -227,6 +231,7 @@ DEFINE
  SMINIMUMBALANCE           = MONEY
  SCURRENTSTATUS            = CHARACTER
  SCURRENTTERM              = CHARACTER
+ SRENEWTERM                = CHARACTER
```

Comment:
I see that SCURRENTTERM gets set but is never referenced.  Should SCURRENTTERM be removed.